legal-api/src/legal_api/services/authz.py

```diff
@@ -306,10 +296,10 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
             },
             'dissolution': {
                 'voluntary': {
-                    'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC', 'SP', 'GP'],
+                    'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC'],
                     'blockerChecks': {
                         'warningTypes': [WarningType.MISSING_REQUIRED_BUSINESS_INFO],
```

Comment:
As SP/GP has been removed from voluntary dissolutions now, we can remove `'warningTypes': [WarningType.MISSING_REQUIRED_BUSINESS_INFO],`